sdk/cosmosdb/arm-cosmosdb/samples-dev/cassandraDataCentersCreateUpdateSample.ts

```diff
@@ -21,10 +21,12 @@ dotenv.config();
  * This sample demonstrates how to Create or update a managed Cassandra data center. When updating, overwrite all properties. To update only some properties, use PATCH.
  *
  * @summary Create or update a managed Cassandra data center. When updating, overwrite all properties. To update only some properties, use PATCH.
- * x-ms-original-file: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2023-03-15-preview/examples/CosmosDBManagedCassandraDataCenterCreate.json
+ * x-ms-original-file: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2023-09-15/examples/CosmosDBManagedCassandraDataCenterCreate.json
  */
 async function cosmosDbManagedCassandraDataCenterCreate() {
-  const subscriptionId = process.env["COSMOSDB_SUBSCRIPTION_ID"] || "subid";
+  const subscriptionId =
```

Comment:
do we need to run the sample generation for updating the ts and js versions from this?